src/makeInfinityAdder.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+  const result = (x) => {
```

Comment:
Consider using a more descriptive name for the `result` function. For example, you could use `adder` to reflect its purpose.